script/Deploy.s.sol

```diff
@@ -117,46 +116,46 @@ contract DeployScript is Script {
             CREATE2_FACTORY
         );
         if (addr != expected) {
-            console2.log("Expected address mismatch");
-            console2.log("Expected: ", expected);
-            console2.log("Actual: ", addr);
+            console.log("Expected address mismatch");
+            console.log("Expected: ", expected);
+            console.log("Actual: ", addr);
             revert();
         }
 
         if (addr.code.length == 0) {
-            console2.log("No code found at expected address, deploying...");
+            console.log("No code found at expected address, deploying...");
             AccountFactory deployed = new AccountFactory{salt: salt}(
                 entryPoint, UpgradeableModularAccount(payable(accountImpl)), singleSignerValidation, owner
             );
 
             if (address(deployed) != expected) {
-                console2.log("Deployed address mismatch");
-                console2.log("Expected: ", expected);
-                console2.log("Deployed: ", address(deployed));
+                console.log("Deployed address mismatch");
+                console.log("Expected: ", expected);
+                console.log("Deployed: ", address(deployed));
                 revert();
             }
 
-            console2.log("Deployed AccountFactory at: ", address(deployed));
+            console.log("Deployed AccountFactory at: ", address(deployed));
         } else {
-            console2.log("Code found at expected address, skipping deployment");
+            console.log("Code found at expected address, skipping deployment");
         }
     }
 
     function _addStakeForFactory(uint32 unstakeDelay, uint256 stakeAmount) internal {
-        console2.log("Adding stake to factory");
+        console.log("Adding stake to factory");
 
         uint256 currentStake = entryPoint.getDepositInfo(address(factory)).stake;
-        console2.log("Current stake: ", currentStake);
+        console.log("Current stake: ", currentStake);
         uint256 stakeToAdd = stakeAmount - currentStake;
 
         if (stakeToAdd > 0) {
-            console2.log("Adding stake: ", stakeToAdd);
-            entryPoint.addStake{value: stakeToAdd}(unstakeDelay);
```

Comment:
: |

Reply:
haha this comment made me realize a mistake - we should use the value `stakeToAdd`, not `stakeAmount`